src/main/java/com/team4/testingsystem/services/TestEvaluationService.java

```diff
@@ -0,0 +1,7 @@
+package com.team4.testingsystem.services;
+
+import com.team4.testingsystem.entities.Test;
+
+public interface TestEvaluationService {
+    int getEvaluationByTest(Test test);
```

Comment:
Can't see where is it used. Please set the evaluation on finish method of test service